### PR TITLE
Wi 48 photo quick view cindy

### PR DESCRIPTION
**Pull request by Cindy088**
get users avatar, username, post photo succeed.

### CODE REVIEW

**Review by JeremyyyLiu — APPROVED** [review]
Fully tested and approve to merge to dev

File: .gitignore [hunk 1]
```diff
@@ -4,6 +4,7 @@
 /node_modules
 /.pnp
 .pnp.js
+.env
```
Comment (Chanran) [inline .gitignore:7]: Add ".DS_Store"  to ".gitignore" file so you don't need to delete ".DS_Store" every time you commit code.

File: next.config.js [hunk 2]
```diff
@@ -4,9 +4,12 @@ module.exports = {
   async rewrites() {
     return [
       {
-        source: '/api/:path*',
-        destination: 'https://api.example.com/:path*',
+        source: "/api/:path*",
+        destination: `${process.env.BACKEND_API}${process.env.BACKEND_PORT}/api/:path*`, // http://localhost:9090/ => api/
```
Comment (Chanran) [inline next.config.js:8]: You should define variables instead of writing code in one line.

File: pages/activated/[token].tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+import Button from "@mui/material/Button";
+import { createTheme, ThemeProvider } from "@mui/material/styles";
+import Box from "@mui/material/Box";
+import Container from "@mui/material/Container";
+import Copyright from "../../src/components/Copyright"
+import axiosRequest from "../../src/utils/axiosRequest";
+
+export default function Activated() {
+    const router = useRouter()
+    // const{token} = router.query.token
```
Comment (Chanran) [inline pages/activated/[token].tsx:15]: delete useless code.

File: pages/activated/[token].tsx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    const { token } = router.query
+
+    let config = {
+        headers: {
+            "Content-Type": "application/json"
+        }
+    }
+
+    axiosRequest(
```
Comment (Chanran) [inline pages/activated/[token].tsx:24]: Move the request to "api.ts" file

File: pages/admin-signin.tsx [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+  const router = useRouter();
+
+  // Use the useForm hook to create a form controller
+  const methods = useForm<IFormInput>({
+    resolver: yupResolver(formSchema),
+  });
+  // Define a submit handler for the form
+  const onSubmit: SubmitHandler<IFormInput> = async (data: IFormInput) => {
+    try {
+      const response = await axiosRequest(
```
Comment (Chanran) [inline pages/admin-signin.tsx:44]: Move request to "api.ts" file

File: pages/admin-signin.tsx [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+        "/api/admin-auth/login",
+        "POST",
+        data
+      );
+      console.log(response);
+      if (response.status === 200) {
+        window.localStorage.setItem("accessToken", response.data.access_token);
```
Comment (Chanran) [inline pages/admin-signin.tsx:51]: you should move all the storage code to "storage.ts" file

File: src/components/AdminManage/Header.tsx [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+import Button from "@mui/material/Button";
+import { useRouter } from "next/router";
+
+//Render header part of admin manager page
+const Header = () => {
+  const router = useRouter();
+
+  //remover access token to redirect to admin sign in page
+  const logOutHandler = () => {
+    window.localStorage.removeItem("access Token");
```
Comment (Chanran) [inline src/components/AdminManage/Header.tsx:12]: “accessToken” or "access Token"？

File: src/components/Copyright.tsx [hunk 8]
```diff
@@ -1,10 +1,9 @@
-
 import Typography from "@mui/material/Typography";
 import MuiLink from "@mui/material/Link";
 
-export default function Copyright(props:any) {
+export default function Copyright(props: any) {
```
Comment (Chanran) [inline src/components/Copyright.tsx:4]: Do not use any.

**Pull request closed without merging**